packages/camera/camera/example/lib/main.dart

```diff
@@ -100,15 +107,16 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
     // App state changed before we got the chance to initialize.
-    if (controller == null || !controller.value.isInitialized) {
+    if ((controller?.value.isInitialized ?? false)) {
       return;
     }
+
+    final CameraController cameraController = controller!;
```

Comment:
If you put this before the early-return, and use `cameraController` in the condition above, you can keep the logic as-is without any additional `!` or `?` or `??`.